src/main/resources/static/views/main/PasswordResetController.js

```diff
@@ -0,0 +1,55 @@
+var PasswordResetController = angular.module('greatStartApp').controller('PasswordResetController',
+    function ($rootScope, $scope, $routeParams, passwordResetService) {
+
+        $scope.closeFpModal = function () {
+            $scope.forgotPassModal.dismiss();
+        };
+
+        $scope.submitEmail = function () {
+            $scope.submitted = true;
+            $scope.message = null;
+            $scope.error = null;
+            var data = passwordResetService.get({email:$scope.email}, function(){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "data" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'data' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AUnusedVariable)